Lib/unittest/runner.py

```diff
@@ -238,6 +238,8 @@ def run(self, test):
                 infos.append("failures=%d" % failed)
             if errored:
                 infos.append("errors=%d" % errored)
+        elif run == 0:
+            self.stream.write("NO TESTS RUN")
```

Comment:
Also matches tense of pytest:
```suggestion
            self.stream.write("NO TESTS RAN")
```


```console
$ pytest
============================================================================ test session starts =============================================================================
platform darwin -- Python 3.11.2, pytest-7.2.1, pluggy-1.0.0
rootdir: /Users/huvankem/github/norwegianblue, configfile: pyproject.toml
plugins: forked-1.4.0, xdist-3.2.0, timeout-2.1.0, respx-0.20.1, cov-4.0.0, repeat-0.9.1, anyio-3.6.2, lazy-fixture-0.6.3, hypothesis-5.49.0
collected 0 items

=========================================================================== no tests ran in 0.02s ============================================================================
```